api/routes.go

```diff
@@ -33,4 +33,6 @@ func (s *server) routes() {
 	metricsApi.HandleFunc("/{account}/instances/{id}/graph", s.GetEC2MetricsURLHandler).Methods(http.MethodGet)
 	// metrics endpoints for ECS services
 	metricsApi.HandleFunc("/{account}/clusters/{cluster}/services/{service}/graph", s.GetECSMetricsURLHandler).Methods(http.MethodGet)
+	// metrics endpoints for RDS services
+	metricsApi.HandleFunc("/{account}/rdsinstances/{id}/graph", s.GetRDSMetricsURLHandler).Methods(http.MethodGet)
```

Comment:
just `rds`? `dbs`? `rdsinstances` is a mouthful

Reply:
yes, it is a mouthful